src/iuse_actor.cpp

```diff
@@ -2909,6 +2911,27 @@ long heal_actor::finish_using( player &healer, player &patient, item &it, hp_par
         }
     }
 
+    // apply healing over time effects
+    float heal_stack = 4 + 2 * healer.get_skill_level( skill_firstaid );
+    if( it.has_flag( "BANDAGES" ) && it.has_flag( "DISINFECTANT" ) ) {
```

Comment:
Maybe you meant `||` instead of `&&` here?

Reply:
Only first aid kits have both flags and they apply both effects with higher intensity to compensate hight ammount of materials.